cmd/apps/certmanager_app.go

```diff
@@ -142,11 +129,7 @@ func MakeInstallCertManager() *cobra.Command {
 }
 
 const CertManagerInfoMsg = `# Get started with cert-manager here:
-# https://docs.cert-manager.io/en/latest/tutorials/acme/http-validation.html
-
-# Check cert-manager's logs with:
-
-kubectl logs -n cert-manager deploy/cert-manager -f`
```

Comment:
Why are we removing this?

Reply:
We try to disencourage people from using logs to debug their installs as they are very spammy and the kubectl event log will help way more as we now log everything in there